person.rb

```diff
@@ -4,14 +4,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- [ ] I can see you initialized the rental in person.rb. However, we need to have a method `add_rental` with (book, date) parameters. Kindly add that!

`# *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`

Reply:
@Rustamxon7 thank you
But I need little bit explanation where I have to add the ```add_rental(book, date)``` method. Should I add in both ```person.rb```  and ```book.rb```? why date parameter?

Reply:
@GutemaG Left message in slack